.github/workflows/package-build.yaml

```diff
@@ -30,9 +30,9 @@ jobs:
 
       - name: Set up Python 3.x
         uses: actions/setup-python@v5
+        id: setup-python
         with:
           python-version: ${{ matrix.python }}
-          cache: 'pip'
```

Comment:
Why did you also drop this?

Reply:
I removed this as it doesn't do anything. We use poetry to install dependencies which installed the dependencies in a different directory to pip.

Reply:
https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#caching-packages